EIPS/eip-7710.md

```diff
@@ -0,0 +1,250 @@
+---
+eip: 7710
```

Comment:
```suggestion
eip: TBA
```
EIP/ERC numbers are assigned by editors/associates.  Please don't self assign, 7710 was already assigned to an ERC.

If this is an ERC (application layer) please close this PR and open a new PR in the ERC repo.